src/ucp/core/ucp_worker.inl

```diff
@@ -130,6 +130,15 @@ ucp_worker_iface_get_attr(ucp_worker_h worker, ucp_rsc_index_t rsc_index)
     return &ucp_worker_iface(worker, rsc_index)->attr;
 }
 
+/**
+ * @return worker's iface attributes by resource index
```

Comment:
maybe mention that these attrs adjusted based on distance, otherwise description is the same as for `ucp_worker_iface_get_attr`